extensions/apps/extensions/src/components/pages/my-extensions/my-extensions.tsx

```diff
@@ -141,7 +141,16 @@ export const MyExtensionsPage: React.FC<unknown> = () => {
   }, [authenticatedDID, showErrorNotification]);
 
   useEffect(() => {
-    getDraftExtensions();
+    const getDraftExtensions = () => {
```

Comment:
why did you redefine getDraftExtensions here? there is still the function defined at line 132